### ISSUE TITLE
Prevent sharing of group folder itself

### ISSUE THREAD

**Issue by opus-nbo** [nextcloud/groupfolders#2045]
I have a question about sharing rights and could not find anything about this situation:

When a group folder is created and assigned to Groups, the group folder itself can be shared. Can this be prevented in any way?

**Comment by JohnWalkerx** [comment]
@opus-nbo Yes, you can prevent it. In the group folder settings where you assign the groups you can uncheck the "share" and the no sharing of the group folder is possible.
But then no sharing in the group folder is possible.

**Comment by opus-nbo** [comment]
@JohnWalkerx thanks, thats the problem. I need a group folder that cannot be shared but sharing of it's contents should be possible.

**Comment by JohnWalkerx** [comment]
@opus-nbo Maybe #2021 is what you need. There is a new setting to disable root sharing.
I didn't tested it. 
Could be that you need to set this setting via occ.

**Comment by icewind1991** [comment]
You can use `occ config:app:set groupfolders allow_root_share --value false` to disable sharing the root of the groupfolder globally, setting this per-folder isn't possible atm
